src/modules/SdnDiag.Health/private/Test-ResourceConfigurationState.ps1

```diff
@@ -25,24 +25,72 @@ function Test-ResourceConfigurationState {
         foreach($object in $sdnResources){
             # examine the provisioning state of the resources and display errors to the screen
             if ($object.properties.provisioningState -ine 'Succeeded') {
+                $msg = "{0} is reporting provisioning state: {1}" -f $object.resourceRef, $object.properties.provisioningState
+                switch ($object.properties.configurationState.Status) {
```

Comment:
Is it necessary to show configuration state if Provisioning State is not Success?

Reply:
Implemented and will skip further validation if provisionState is not Success